File: TGME/src/main/java/tmge/model/GameInfo.java
package tmge.model;

import javafx.scene.image.Image;

public class GameInfo {
    private String name;
    private String description;
    private Image thumbnail;

    public GameInfo(String name, String description, String thumbnailPath) {
        this.name = name;
        this.description = description;
        this.thumbnail = new Image(thumbnailPath);
    }

    public String getName() {
        return name;
    }

    public String getDescription() {
        return description;
    }

    public Image getThumbnail() {
        return thumbnail;
    }
}

File: TMGE/registry/game_info.py
class GameInfo:
    def __init__(self, name: str, description: str, thumbnail=None):
        self.name = name
        self.description = description
        self.thumbnail = thumbnail
    
    def get_name(self) -> str:
        return self.name
    
    def get_description(self) -> str:
        return self.description
    
    def get_thumbnail(self):
        return self.thumbnail